tools/bazel-test-gdb

```diff
@@ -12,7 +12,7 @@
 if [[ ! "$1" =~ (@[a-zA-Z0-9_-]+)?//.*:[a-zA-Z0-9_-]+ ]]
 then
   echo "First argument to $0 must be a [@repo]//test/foo:bar label identifying a single test to run"
-  echo "$1 does not mach this pattern"
+  echo "$1 does not match this pattern"
```

Comment:
Probably worth doing in a separate one line PR, to keep the reviewer's job a little easier and the commit log for the project cleaner.